src/docs/product/accounts/sso/okta-sso/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
this is renamed from `src/docs/product/accounts/sso/okta-sso.mdx` -- the only difference in the file is i've removed the SCIM section at the bottom.